examples/deployAccountUDC/main.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math/big"
+	"math/rand"
+	"os"
+
+	"github.com/NethermindEth/juno/core/felt"
+	"github.com/NethermindEth/starknet.go/account"
+	"github.com/NethermindEth/starknet.go/rpc"
+	"github.com/NethermindEth/starknet.go/utils"
+	"github.com/joho/godotenv"
+)
+
+// NOTE : Please add in your keys only for testing purposes, in case of a leak you would potentially lose your funds.
+var (
+	name                  string = "testnet"                                                            // env."name"
+	account_addr          string = "0x06f36e8a0fc06518125bbb1c63553e8a7d8597d437f9d56d891b8c7d3c977716" // Replace it with your account address
+	account_cairo_version        = 0                                                                    // Replace  with the cairo version of your account
+	privateKey            string = "0x0687bf84896ee63f52d69e6de1b41492abeadc0dc3cb7bd351d0a52116915937" // Replace it with your account private key
+	public_key            string = "0x58b0824ee8480133cad03533c8930eda6888b3c5170db2f6e4f51b519141963"  // Replace it with your account public key
```

Comment:
suggestion: 
1. move all variables that need to be replaced in .env.template file (all values are set with comment to replace it user .env file)
2. add .gitignore in this directory with this content:
```
.env.*
!.env.template
```

this way user don't have to modify anything in code and is safe for comitting private data to some repo

Reply:
Mind it I push this to a separate issue that we can include in the next release (all the examples will need updated)?

Reply:
Sure